model-optimizer/setup.py

```diff
@@ -111,8 +111,7 @@ def find_package_modules(self, package, package_dir):
       'kaldi': read_text('requirements_kaldi.txt'),
       'mxnet': read_text('requirements_mxnet.txt'),
       'onnx': read_text('requirements_onnx.txt'),
-      'tensorflow': read_text('requirements_tf.txt'),
-      'tensorflow2': read_text('requirements_tf2.txt'),
+      'tensorflow2': read_text('requirements_tf.txt'),
```

Comment:
Inconsistency between the name and the configuration file. Maybe keep both keys "tensorflow" and "tensorflow2" but install requirements from the same file?

Reply:
Done.